src/main/java/TaskList.java

```diff
@@ -0,0 +1,109 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    protected static ArrayList<Task> MANAGER;
```

Comment:
There is no need for the `MANAGER` to be capital. Only final constants are supposed to be all capital.